src/Plugin/views/query/Elasticsearch.php

```diff
@@ -540,14 +540,7 @@ protected function getAllEntities() {
    * @return int
    */
   protected function getTotalHits($data) {
-    if (ElasticsearchClientVersion::getMajorVersion() >= 7) {
-      $total_items = isset($data['hits']['total']['value']) ? $data['hits']['total']['value'] : 0;
-    }
-    else {
-      $total_items = isset($data['hits']['total']) ? $data['hits']['total'] : 0;
-    }
-
-    return $total_items;
+    return $data['hits']['total']['value'] ?? $data['hits']['total'] ?? 0;
```

Comment:
Won't this cause notices on undefined indexes though? Or does the ternary operator imply an "isset" check on the variables somehow?